superchain/configs/sepolia/metal.yaml

```diff
@@ -0,0 +1,25 @@
+name: Metal L2 Testnet
+chain_id: 1740
+public_rpc: https://testnet.rpc.metall2.com
+sequencer_rpc: https://testnet.rpc.metall2.com
+explorer: https://testnet.explorer.metall2.com
+
+superchain_level: 1
+standard_chain_candidate: true # This is a temporary field which causes most of the standard validation checks to run on this chain
+superchain_time: null # Missing hardfork times are NOT yet inherited from superchain.yaml
+contracts_version_tag: "op-contracts/v1.3.0" # Multi-Chain Prep (MCP) https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.3.0
```

Comment:
From @bitwiseguy : Do we need to harmonize the quoting of strings in this file with what we are doing in other yaml files and in our check-rollup-config command?